components/TxForm.tsx

```diff
@@ -20,7 +20,7 @@ import {
   deserializeTransaction,
 } from "@railgun-community/shared-models";
 import { erc20ABI } from "@wagmi/core";
-import { ethers, constants } from "ethers";
+import { ethers, constants, BigNumber } from "ethers";
```

Comment:
_(arbitrary comment location to start a thread)_

When I run `yarn` I get a different lockfile, mind updating the lockfile? Think it's just an artifact of the two open PRs that were merge prior to this having different lockfiles

Reply:
Weird, yes I will do that

Reply:
I'm still reproducing this, so yea I think that's ok to do

Reply:
Deleted and regenerated, hopefully this resolves those issues

Reply:
Regenerated now

Reply:
Huh I'm still getting a different lockfile when I run `yarn`. @wildmolasses any thoughts?

Reply:
Can you post what the yarn.lock changes are? Maybe a pr against this branch.

Reply:
are you guys using the same version of yarn? are you both using volta?

Reply:
ah yes I was on the wrong yarn version, thanks!